src/pages/settings/Subscription/SubscriptionSize/utils.ts

```diff
@@ -0,0 +1,18 @@
+import {addMonths, format, startOfMonth} from 'date-fns';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import type ONYXKEYS from '@src/ONYXKEYS';
+import INPUT_IDS from '@src/types/form/SubscriptionSizeForm';
+
+const validate = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.SUBSCRIPTION_SIZE_FORM>): FormInputErrors<typeof ONYXKEYS.FORMS.SUBSCRIPTION_SIZE_FORM> => {
+    const errors = ValidationUtils.getFieldRequiredErrors(values, [INPUT_IDS.SUBSCRIPTION_SIZE]);
+    if (values[INPUT_IDS.SUBSCRIPTION_SIZE] && !ValidationUtils.isNumeric(values[INPUT_IDS.SUBSCRIPTION_SIZE])) {
+        errors.subscriptionSize = 'subscriptionSize.error.size';
+    }
+
+    return errors;
+};
+
+const getNewSubscriptionRenewalDate = (): string => format(startOfMonth(addMonths(new Date(), 11)), 'MMM d, yyyy');
```

Comment:
```suggestion
const getNewSubscriptionRenewalDate = (): string => format(startOfMonth(addMonths(new Date(), 11)), CONST.DATE.MONTH_DAY_YEAR_ABBR_FORMAT);
```

Reply:
I think we will need this method on other screens too 

wdyt?

Reply:
Dont think so. In majority of the cases this date will come from ONYX, we will just have to format this. This helper prepares new date when editing subscription size

Reply:
Yeah, if it's going to be used in other places it's a good candidate to be on `SubscriptionUtils` for example

Reply:
if such need arises then we can move it, is it ok to leave this here for now?

Reply:
> if such need arises then we can move it, is it ok to leave this here for now?

Up to you @pasyukevich 😉

Reply:
we can leave it here and move it up when it is needed